File: elementos.py
def elemento(z):
    if z <= 2:
        return "Gas noble"
    elif 3 <= z <= 10:
        return "No metal"
    elif 11 <= z <= 18:
        return "Metal alcalino"
    elif 19 <= z <= 36:
        return "Metaloide"
    else:
        return "Elemento pesado"


z = int(input("Número de Quintaesencia del elemento: "))

# Llamamos a la función y mostramos el resultado
print("La categoría mística del elemento es:", elemento(z))

File: elementos.cpp
#include <iostream>  // Biblioteca estándar para entrada y salida
using namespace std;

string clasificarElemento(int z) {
    if (z <= 2) {  
        return "Gas noble";
    } else if (z >= 3 && z <= 10) {  
        return "No metal";
    } else if (z >= 11 && z <= 18) {  
        return "Metal alcalino o alcalinotérreo";
    } else if (z >= 19 && z <= 36) {  
        return "Metaloide o no metal";
    } else {  
        return "Elemento pesado";
    }
}

int main() {
    int z;  


    cout << "Ingrese el Número de Quintaesencia: ";
    cin >> z;

 
    cout << "Categoría mística del elemento: " << clasificarElemento(z) << endl;

    return 0;  
